src/Generator/AtomGenerator.php

```diff
@@ -120,7 +120,7 @@ private function addNode( DOMDocument $dom, DOMElement $head, $name, $value, $ty
 			return;
 		}
 
-		$node = $dom->createElement( $name, $value );
+		$node = $dom->createElement( $name, str_replace( '&', '&amp;', $value ) );
```

Comment:
Actually, I think another way to do this might be to set `textContent` directly, e.g.:

```php
		$node = $dom->createElement( $name );
		$node->textContent = $value;
```

I'm not sure if there are other considerations, so probably it's fine to handle the one-off case you've found.

Reply:
It sounds much better. Thank you!  I create #398 to do it.